hooks/build

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+docker build \
```

Comment:
This hook behavior is undocumented other than as a comment in a GitHub issue (https://github.com/docker/hub-feedback/issues/508#issuecomment-240616319). Seems to work though.